docs/build/manifest.md

```diff
@@ -5,39 +5,86 @@ The Manifest `project.yaml` file can be seen as an entry point of your project a
 The Manifest can be in either YAML or JSON format. In this document, we will use YAML in all the examples. Below is a standard example of a basic `project.yaml`.
 
 <CodeGroup>
+
+
+<CodeGroupItem title="v1.0.0" active>
+
+``` yml
+
+specVersion: 1.0.0
+name: subquery-starter
+version: 0.0.4
+runner:
+  node:
+    name: '@subql/node'
+    version: ^0.33.0
```

Comment:
Should this be `latest`?

Reply:
Okay, we can use `latest` for one of the version, the purpose here is to let user understand they can point to a specify version